library/ripemd160.c

```diff
@@ -460,6 +495,12 @@ int mbedtls_ripemd160_self_test( int verbose )
         mbedtls_printf( "\n" );
 
     return( 0 );
+
+fail:
+    if( verbose != 0 )
+        mbedtls_printf( "failed\n" );
+
+    return( ret );
 }
```

Comment:
I am saying that while we document our self-tests to always return `0` or `1`, we do not consistently adhere to this - sometimes we do return specific error codes, sometimes we don't even if they're available.